lib/bankaccounts/account.rb

```diff
@@ -0,0 +1,37 @@
+module BankAccounts
+
+  class Account
+    attr_accessor :id, :balance
+
+    def initialize(id, initial_balance)
+      # raises an error if initial_balance is negative
+      if initial_balance < 0
+        raise ArgumentError.new("Only positive initial balances allowed")
```

Comment:
:+1: Alternatively, you can also just say `raise ArgumentError, "Only positive ..."` instead of actually creating an instance of ArgumentError.